crates/storage/store/storage.rs

```diff
@@ -698,6 +698,22 @@ fn hash_key(key: &H256) -> Vec<u8> {
         .to_vec()
 }
 
+// USED ONLY FOR TESTS.
```

Comment:
I would remove this from this PR and create a separate ticket to see if we want to tackle this.

It seems every crate has a different way of setting up a store for tests, and if we decide to create a helper to abstract this boilerplace, it should be done across the board.

I would keep this PR focused on the feature instead.

https://github.com/search?q=repo%3Alambdaclass%2Flambda_ethereum_rust+%22Store%3A%3Anew%22&type=code

Reply:
But without this specific function; I cannot test the feature properly since I did not find a way to simulate the addition of blocks between RPC calls, how can I test it then? should I skip testing it for now?

Reply:
Ah, I saw the test now. I would not add such a high level test here, it looks like a full fledged integration test.

Personally, I would just have unit tests inside the rust modules. If you have trouble testing the functionality, it may mean that you should move the business logic to a separate module (I think we discussed this at some point). For integration tests, we could leverage something like hive (we have our own fork) to write custom tests, or we could build or own framework.

Anyway, since these endpoints are not high priority right now, I would just eliminate that test, and create a ticket to work on this at some point.

Reply:
> it may mean that you should move the business logic to a separate module (I think we discussed this at some point)
 
 I can't remember if we did, but I'll keep in in mind for the future.
 
 As per the PR:
 - I've removed the test.
 - I've removed the function from the storage crate.
 - [ I've added an issue to eventually test this endpoint.](https://github.com/lambdaclass/lambda_ethereum_rust/issues/965)